database/queries/get_all_careers.js

```diff
@@ -0,0 +1,15 @@
+const dbConnection = require('../db_connection');
+
+const getAllCareers = callback => {
+  const allCareersQuery = 'SELECT * FROM careers;';
+  dbConnection.query(allCareersQuery, (error, response) => {
+    if (error) {
+      return callback(error);
+    } else {
+      console.log(response);
+      callback(null, response);
```

Comment:
You could destructure/return the actual rows rather than the whole response object.

Reply:
yeah I thought about that but I wasn't sure what I would get back from the database so I left it for now

Reply:
Can change though :man_shrugging: